contracts/interfaces/IStrategyBase.sol

```diff
@@ -1,17 +1,43 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.7.0;
 
+/// @title IStrategyBase Interface
+/// @author imToken Labs
 interface IStrategyBase {
+    /// @notice Emitted when spender address is updated
```

Comment:
```suggestion
    /// @notice Emitted when Tokenlon spender address is updated
```

Reply:
Adjusted in 7a91187197c37a8285b47ee4f3227bad434ebccb.